.circleci/circle.yml

```diff
@@ -0,0 +1,134 @@
+version: 2
```

Comment:
Did you pull this from another repo? Guessing there was an example you were working from?

Reply:
Correct - some of the repos have already been updated. For example: https://github.com/segment-integrations/analytics-android-integration-firebase/blob/master/.circleci/config.yml